tools/CoqMakefile.in

```diff
@@ -362,7 +363,7 @@ post-all::
 	@# Extension point
 .PHONY: post-all
 
-real-all: $(VOFILES) $(if $(USEBYTE),bytefiles,optfiles)
+real-all: $(VDFILES) $(VOFILES) $(if $(USEBYTE),bytefiles,optfiles)
```

Comment:
I think you don't need this dependency; it's picked up automatically because the makefile `-include`s the `*.v.d` files.

Reply:
I was not aware of this trick with `-include`, thanks.